content/joysticks/winwing/winwing-cdu/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: WINWING CDU
+weight: 50
+---
+
+WINWING CDU display support is automatic on run, if a supported plane and a CDU are detected. To prevent MobiFlight from taking over the CDU, disable it in `Extras->Settings->Peripherals`.
+
+## Installing Python
+
+Follow the [guide](/guides/installing-python/).
+
+## General remarks
+
+- Python is a mandatory prerequisite.
+- If MobiFlight has detected the CDU on startup, it will show `->MobiFlight<-` on the display.
+- Do not run SimAppPro in parallel. After using SimAppPro, reconnect device to the USB port. MobiFlight currently relies on the CDU default power-on display settings, which might have been changed by a running  SimAppPro instance.
```

Comment:
I've read this a few times and don't understand it. Is there a way to simplify this, or to re-order it so it shows what is required first, and then the thing to do second?